app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached = {}
+
+    def inner(*args, **kwargs) -> Callable:
+        key = f"{func.__name__} {str(args)} {str(kwargs)}"
```

Comment:
You just can check if args already in your cache storage , you dont need to create key variable